src/RetireNet.Packages.Tool/Services/RetireServiceOptions.cs

```diff
@@ -9,5 +9,9 @@ public class RetireServiceOptions
         public string Path { get; set; }
 
         public bool AlwaysExitWithZero { get; set; }
+        public string ReportPath { get; set; }
+        public String ReportFormat { get; set; }
+        public bool NoRestore { get; set; }
```

Comment:
Sounds a bit out of scope of solving the reporting feature? Do you have any other reason for this to be added, please create a new issue and let's discuss it in a sep PR.

My 2 cents: I don't think this adds any new value, and can be removed. The tool only restores if it does not find an already restored project (an existing `projects.assets.json` means a restore does not happen).

Reply:
I need the 'no restore' in order to get it to run on at least the first project I ran it on, within our build process. I didn't (and still don't, really) have time to figure out why, but telling it not to run restore did the trick, and this was running right after `dotnet restore` so the projects.assets.json 'tell' might be something something configurable that my team has configured into something else. I dunno. I'm not a C# guy by history, only by recent trade. :)